### PR TITLE
feat(trace-viewer): add request method/status to the network details tab

### PR DESCRIPTION
**Pull request by ryanrosello-og**
### Add Request Method and Request Status to the Request details tab of the Trace Viewer

Currently when viewing a particular request in the Trace Viewer, the Request details tab only shows the Request URL and Request Headers. It would be helpful to also show the **Request Method** and **Request Status** in this tab. This would make it easier to understand the request details without having to click the (X) button to view the request table which shows the Request Method and Request Status.

**Before:**

**After:**

### CODE REVIEW

**Comment by ryanrosello-og** [comment]
@microsoft-github-policy-service agree

File: packages/trace-viewer/src/ui/networkResourceDetails.tsx [hunk 1]
```diff
@@ -79,6 +79,12 @@ const RequestTab: React.FunctionComponent<{
   return <div className='network-request-details-tab'>
     <div className='network-request-details-header'>URL</div>
     <div className='network-request-details-url'>{resource.request.url}</div>
+    <div className='network-request-details-header'>Method</div>
```
Comment (yury-s) [inline packages/trace-viewer/src/ui/networkResourceDetails.tsx:82]: Can we reduce repetition ( Method Method:, Status Status:), perhaps put non header details into one `General` section and make `Request URL:`, `Method:` and `Status:` individual lines under it?

Comment (ryanrosello-og) [reply packages/trace-viewer/src/ui/networkResourceDetails.tsx]: All done @yury-s

File: packages/trace-viewer/src/ui/networkResourceDetails.tsx [hunk 2]
```diff
@@ -77,8 +77,12 @@ const RequestTab: React.FunctionComponent<{
   }, [resource]);
 
   return <div className='network-request-details-tab'>
-    <div className='network-request-details-header'>URL</div>
-    <div className='network-request-details-url'>{resource.request.url}</div>
+    <div className='network-request-details-header'>General</div>
+    <div className='network-request-details-url'>{`URL: ${resource.request.url}`}</div>
+    <div className='network-request-details-general'>{`Method: ${resource.request.method}`}</div>
+    <div className='network-request-details-general'>{`Status Code: ${resource.response.status >= 200 && resource.response.status < 400
```
Comment (dgozman) [inline packages/trace-viewer/src/ui/networkResourceDetails.tsx:83]: Let's also show `resourse.response.statusText`? Something like `Status Code: 200 OK`.

Comment (ryanrosello-og) [reply packages/trace-viewer/src/ui/networkResourceDetails.tsx]: I've added the `resourse.response.statusText` @dgozman , @yury-s

**Review by dgozman — APPROVED** [review]
Thank you for the PR! Looks good, merging in.

**Pull request merged**